Public/Src/Engine/Processes/FileAccessManifest.cs

```diff
@@ -421,6 +421,15 @@ public void UnsetMessageCountSemaphore()
             m_messageCountSemaphoreName = null;
         }
 
+        /// <summary>
+        /// Optional AnyBuild shim execution information.
+        /// </summary>
+        /// <remarks>
+        /// Internal since this is set as a side effect of initializing a FileAccessManifest
+        /// in <see cref="SandboxedProcessInfo"/>.
+        /// </remarks>
+        internal AnyBuildShimInfo AnyBuildShimInfo { get; set; }

```

Comment:
>AnyBuildS [](start = 17, length = 9)

Isn't this more generically applicable? Should we brand it?

Reply:
Espcially since Cpp side uses a different naming: `ShimProcess`

---
In reply to: [282950720](https://github.com/microsoft/BuildXL/pull/279#discussion_r282950720) [](ancestors = 282950720)